lang/cs/Org.Apache.REEF.Utilities/Org.Apache.Reef.Utilities.DotNet.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Org.Apache.Reef.Utilities</AssemblyName>
+    <OutputPath>..\bin\$(AssemblyName)</OutputPath>
+    <TargetFrameworks>netstandard2.0;net46;net451</TargetFrameworks>
+    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard2.0' ">2.0.0-beta-25123-01</NetStandardImplicitPackageVersion>
+    <DefaultItemExcludes>$(DefaultItemExcludes);**/AssemblyInfo.cs</DefaultItemExcludes>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.2-beta2" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="Utils\Templates\ClassTemplate.tt">
```

Comment:
I removed. These are artifacts from older dotnet cli tools and no longer needed.